c/rce_validator.c

```diff
@@ -92,27 +92,35 @@ static int make_witness_cursor(uint8_t* buffer, uint32_t cache_size,
 }
 
 int main() {
+  // Load Script args first.
+  //
+  // | Script Args   | Type ID | Flags |
+  // | ------------- |---------| ------|
+  // | Length(Byte)  | 32      | 1     |
+
+  uint8_t current_script[32768];
+  uint64_t len = 32768;
+  int err = ckb_checked_load_script(current_script, &len, 0);
+  CHECK(err);
+  CHECK2(len <= 32768, ERROR_SCRIPT_TOO_LONG);
```

Comment:
If the checked version (ckb_checked_load_script) is used, it's not needed to check "len" again here.

Reply:
It's wired, you wrote that too 

https://github.com/nervosnetwork/ckb-production-scripts/blob/xudt_rce/c/xudt_rce.c#L231-L233